airbyte-integrations/connectors/destination-gcs/src/main/resources/spec.json

```diff
@@ -12,31 +12,32 @@
     "required": [
       "gcs_bucket_name",
       "gcs_bucket_path",
-      "gcs_bucket_region",
       "credential",
```

Comment:
Can we please make sure that *every field* explicitly has
- '(Optional)' or and asterisks denoting it's an optional or required field 
- Links to documentation for additional information on what the field is and how to find it